internal/ratelimit/gubernator/gubernator.proto

```diff
@@ -0,0 +1,189 @@
+/*
+Copyright 2018-2019 Mailgun Technologies Inc
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+syntax = "proto3";
+
+option go_package = "gubernator";
+
+option cc_generic_services = true;
+
+package pb.gubernator;
+
+import "google/api/annotations.proto";
+
+service V1 {
+
+  // Given a list of rate limit requests, return the rate limits of each.
+  rpc GetRateLimits (GetRateLimitsReq) returns (GetRateLimitsResp) {
+    option (google.api.http) = {
+      post: "/v1/GetRateLimits"
+      body: "*"
+    };
+  }
+
+
+  // This method is for round trip benchmarking and can be used by
+  // the client to determine connectivity to the server
+  rpc HealthCheck (HealthCheckReq) returns (HealthCheckResp) {
+    option (google.api.http) = {
+      get: "/v1/HealthCheck"
+    };
+  }
+}
+
+// Must specify at least one Request
+message GetRateLimitsReq {
```

Comment:
```suggestion
message GetRateLimitsRequest {
```
I would make it full word and below. Code is generated anyway (See https://developers.google.com/protocol-buffers/docs/style)

Reply:
Unless you copied this code (if so, let's comment about this)

Reply:
This code piece is copied directly from gubernator repo. The script that copies it in Makefile. I will add a README to the directory.